00-strings/00-strings.md

```diff
@@ -0,0 +1,24 @@
+# strings
+
+Welcome to strings exercise!
+
+## Verifying your work
+
+Run `python3 test_strings.py` to confirm that you have succesfully created the required functions.
```

Comment:
I'm not sure if this would be out of scope, but `virtualenv` is a much better way to control Python versions, instead of using `python2` or `python3`.

Reply:
I think that we then would need to clarify that what tool should be used 😅 

I personally like virtualenv + wrapper and use it but then I also know people who are saying that pyenv is the #1 or the pipenv

Reply:
I'm also using https://virtualenvwrapper.readthedocs.io/en/latest/index.html Whatever tool you pick, you'll get a various comments from people who may be familiar with one tool, instead of the other. The combination of `virtualenv` with/without `virtualenvwrapper` is probably the most used/common choice. My argument is just that teaching people a basic thing: a Python project can have its own environment, with a set of packages in defined versions.

Reply:
@atombrella should we create issue from the virtualenv/some other tooling? atm I'm not 100% sure that where that documentation should be added 😅

Reply:
Yes. Let's just create an issue :)